### DISCUSSION TITLE
Can't sign into Github Copilot in intellij over corporate network

### DISCUSSION THREAD

**Discussion by Pasha-Akito** [community/community#33245] [Copilot Conversations]
Hey, would anyone have a workaround or fix for this? Got a month of copilot and would rather not waste it not being able to use it. I'm using intellij 2021.2.3 and have copilot 1.1.31.1908 plugin installed. 
![image](https://user-images.githubusercontent.com/55661677/190598462-18658aff-27bf-44ed-8d4d-ce3f87899bae.png)

**Comment by iamcjwilliams** [comment]
@Pasha-Akito Are you still experiencing this issue?

**Reply by Pasha-Akito** [reply]
Yes still experiencing this issue

**Reply by iamcjwilliams** [reply]
Are you experiencing the issue if you use copilot outside of the corporate network?

**Reply by Pasha-Akito** [reply]
Yes I can use it on my home machine just fine. can't use it on the work laptop since it's always connected to the network

**Reply by iamcjwilliams** [reply]
@Pasha-Akito It sounds like there may be a firewall through your corporate network, you may need to get in touch with your company's IT team

**Reply by Pasha-Akito** [reply]
I might just do that actually, It has been approved by the IT team but I'll hop on call with someone to figure out how to make it work

**Reply by AlexFHer** [reply]
i'm also having this same issue, I've contacted the IT team and they had a look but could not find anything, they just dont know which access shall they grant or ip. If someone knows which ip or service it is would be nice.

**Reply by Pasha-Akito** [reply]
hopefully there's a fix or workaround soon

**Comment by priyojitsingha2001** [comment]
I am also facing this issue but lot of my team members who are using the older version of the plugin are using it as usual. is there any way through which we can fall back to the older version by maybe replacing the file in root directry
